clients/python-wrapper/lakefs/__main__.py

```diff
@@ -155,16 +155,12 @@ def run_binary(binary_path: str, args: list[str]):
     '''
     Run the binary with the provided arguments.
     '''
-    if not binary_path:
-        raise RuntimeError("binary not found")
-    binary_name = os.path.basename(binary_path)
+    print(f'running {binary_path}...')
     try:
-        print(f'running {binary_path}...')
-        proc = subprocess.run(
-            [binary_path] + args, check=False, env=os.environ)
-        return proc.returncode
-    except subprocess.CalledProcessError as e:
-        raise RuntimeError(f"Error executing {binary_name}: {e}") from e
+        proc = subprocess.run([binary_path, *args], check=False)
```

Comment:
We could also use [`os.execvp`](https://docs.python.org/3/library/os.html#os.execvp) on posix systems.

I see that `ruff` does that, which also has a python wrapper executing rust binary.

https://github.com/astral-sh/ruff/blob/d08e41417971f1d05b9daa75f794536a1dd4bedf/python/ruff/__main__.py#L88